chainregistry.go

```diff
@@ -22,23 +22,45 @@ import (
 	"github.com/lightningnetwork/lnd/keychain"
 	"github.com/lightningnetwork/lnd/lnwallet"
 	"github.com/lightningnetwork/lnd/lnwallet/btcwallet"
+	"github.com/lightningnetwork/lnd/lnwire"
 	"github.com/lightningnetwork/lnd/routing/chainview"
 	"github.com/roasbeef/btcd/chaincfg/chainhash"
 	"github.com/roasbeef/btcd/rpcclient"
+	"github.com/roasbeef/btcutil"
 	"github.com/roasbeef/btcwallet/chain"
 	"github.com/roasbeef/btcwallet/walletdb"
 )
 
+const (
+	defaultBitcoinMinHTLCMSat   = lnwire.MilliSatoshi(1000)
+	defaultBitcoinBaseFeeMSat   = lnwire.MilliSatoshi(1000)
+	defaultBitcoinFeeRate       = lnwire.MilliSatoshi(1)
+	defaultBitcoinTimeLockDelta = 144
+	defaultBitcoinStaticFeeRate = lnwallet.SatPerVByte(50)
+
+	defaultLitecoinMinHTLCMSat   = lnwire.MilliSatoshi(1000)
+	defaultLitecoinBaseFeeMSat   = lnwire.MilliSatoshi(1000)
+	defaultLitecoinFeeRate       = lnwire.MilliSatoshi(1)
+	defaultLitecoinTimeLockDelta = 576
+	defaultLitecoinStaticFeeRate = lnwallet.SatPerVByte(200)
+	defaultLitecoinMinRelayFee   = btcutil.Amount(1000)
+)
+
 // defaultChannelConstraints is the default set of channel constraints that are
 // meant to be used when initially funding a channel.
 //
 // TODO(roasbeef): have one for both chains
 // TODO(halseth): make configurable at startup?
-var defaultChannelConstraints = channeldb.ChannelConstraints{
+var defaultBtcChannelConstraints = channeldb.ChannelConstraints{
```

Comment:
`godoc` comment above no longer matches. Also missing a `godoc` comment below in for the new ltc channel constraints.